src/pipecat/services/anthropic.py

```diff
@@ -82,12 +87,16 @@ def __init__(
             model: str = "claude-3-5-sonnet-20240620",
             max_tokens: int = 4096,
             enable_prompt_caching_beta: bool = False,
```

Comment:
I believe we should add `max_tokens` and `enable_prompt_caching_beta` to `InputParams` as well. It's a breaking change, but we are breaking a few things in this next release, so it should be fine. In the emotion Cartesia PR we are also breaking it.

Reply:
I held off because this was breaking, but I'll make the change now.